src/Server.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Io\IniUtil;
+use React\Http\Middleware\LimitConcurrentRequestsMiddleware;
+use React\Http\Middleware\RequestBodyBufferMiddleware;
+use React\Http\Middleware\RequestBodyParserMiddleware;
+use React\Socket\ServerInterface;
+
+/**
+ * Facade around StreamingServer with sane defaults for 80% of the use cases.
+ * The requests passed to your callable are fully compatible with PSR-7 because
+ * the body of the requests are fully buffered and parsed, unlike StreamingServer
+ * where the body is a raw ReactPHP stream.
+ *
+ * Wraps StreamingServer with the following middleware:
+ * - LimitConcurrentRequestsMiddleware
+ * - RequestBodyBufferMiddleware
+ * - RequestBodyParserMiddleware (only when enable_post_data_reading is true (default))
+ * - The callable you in passed as first constructor argument
+ *
+ * All middleware use their default configuration, which can be controlled with
+ * the the following configuration directives from php.ini:
+ * - upload_max_filesize
+ * - post_max_size
+ * - max_input_nesting_level
+ * - max_input_vars
+ * - file_uploads
+ * - max_file_uploads
+ * - enable_post_data_reading
+ */
+final class Server
+{
+    const MAXIMUM_CONCURRENT_REQUESTS = 100;
```

Comment:
Minor nitpick: Should probably be `@internal` :+1:

Reply:
Good point, added that :+1: